localSetup/projects/nginx/nginx-conf.d/edlib-api.conf

```diff
@@ -15,79 +15,90 @@ server {
     access_log   off;
 
     location /recommendations {
-     proxy_pass  http://proxy-recommendation;
+     set $upstream  http://proxy-recommendation;
+     proxy_pass              $upstream;
      proxy_set_header        Host            $host;
      proxy_set_header        X-Real-IP       $remote_addr;
      proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
    }
 
     location /resources {
-     proxy_pass  http://proxy-resource;
+     set $upstream  http://proxy-resource;
+     proxy_pass              $upstream;
      proxy_set_header        Host            $host;
      proxy_set_header        X-Real-IP       $remote_addr;
      proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
    }
 
     location /dokus {
-     proxy_pass  http://proxy-doku;
+     set $upstream  http://proxy-doku;
+     proxy_pass              $upstream;
      proxy_set_header        Host            $host;
      proxy_set_header        X-Real-IP       $remote_addr;
      proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
    }
 
     location /auth {
-     proxy_pass  http://proxy-auth;
+     set $upstream  http://proxy-auth;
+     proxy_pass              $upstream;
      proxy_set_header        Host            $host;
      proxy_set_header        X-Real-IP       $remote_addr;
      proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
    }
 
     location /lti {
-     proxy_pass              http://proxy-lti;
+     set $upstream  http://proxy-lti;
+     proxy_pass              $upstream;
      proxy_set_header        Host            $host;
      proxy_set_header        X-Real-IP       $remote_addr;
      proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
      proxy_set_header        X-Forwarded-Proto $scheme;
    }
 
     location /url {
-     proxy_pass              http://edlibapi-url;
+     set $upstream  http://edlibapi-url;
+     proxy_pass              $upstream;
      proxy_set_header        Host            $host;
      proxy_set_header        X-Real-IP       $remote_addr;
      proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
      proxy_set_header        X-Forwarded-Proto $scheme;
    }
 
     location /doku-lti-viewer {
-     proxy_pass  http://doku-lti-viewer:3000;
+     set $upstream  http://doku-lti-viewer:3000;
+     proxy_pass              $upstream;
      proxy_set_header        Host            $host;
      proxy_set_header        X-Real-IP       $remote_addr;
      proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
    }
 
     location /iframe/sockjs-node {
-	    proxy_pass http://proxy-iframe:3000;
+       set $upstream  http://proxy-iframe:3000;
```

Comment:
Each of the incorrectly formatted files are already a mishmash of tabs and spaces and seemingly no consistent tab width. Since these changes are consistent with being inconsistent, I will leave as is. Reformatting can be done in another PR.

Reply:
I think it is better to just fix it. There is no point in leaving something unfixed.